crates/shared/src/price_estimation.rs

```diff
@@ -106,6 +114,18 @@ pub struct Arguments {
     /// The API endpoint for the Balancer SOR API for solving.
     #[clap(long, env)]
     pub balancer_sor_url: Option<Url>,
+
+    /// The trade simulation strategy to use for supported price estimators. This ensures that
+    /// the proposed trade calldata gets simulated, thus avoiding invalid calldata mistakenly
+    /// advertising unachievable prices when quoting, as well as more robustly identifying
+    /// unsupported tokens.
+    #[clap(long, env)]
+    pub trade_simulator: Option<TradeValidatorKind>,
+
+    /// Flag to enable saving Tenderly simulations in the dashboard for failed trade simulations.
+    /// This helps debugging reverted quote simulations.
+    #[clap(long, env)]
+    pub tenderly_save_failed_trade_simulations: bool,
```

Comment:
This setting is missing in the `Display` implementation of the `Arguments` struct.